mx-tester.yml

```diff
@@ -7,7 +7,7 @@ up:
     - until psql postgres://mjolnir-tester:mjolnir-test@127.0.0.1:8083/mjolnir-test-db -c ""; do echo "Waiting for psql..."; sleep 1s; done
     # Launch the reverse proxy, listening for connections *only* on the local host.
     - docker run --rm --network host --name mjolnir-test-reverse-proxy -p 127.0.0.1:8081:80 -v $MX_TEST_CWD/test/nginx.conf:/etc/nginx/nginx.conf:ro -d nginx
-    - yarn install
+    - corepack yarn install
     - npx ts-node src/appservice/cli.ts -r -u "http://host.docker.internal:9000"
```

Comment:
Isn't npx some npm command? Maybe this should also be `corepack yarn dlx`?

- https://yarnpkg.com/cli/dlx